ResearchKit/Common/ORKAxisView.m

```diff
@@ -0,0 +1,147 @@
+/*
+ Copyright (c) 2015, Apple Inc. All rights reserved.
```

Comment:
Add another line with your name or your institution's name:
e.g. ` Copyright (c) 2015, Your Name. All rights reserved.`

Apply this to all the files which has 10+ lines code from your contribution.